t/apicast-syslog.t

```diff
@@ -0,0 +1,57 @@
+use lib 't';
+use Test::APIcast::Blackbox 'no_plan';
+
+repeat_each(3);
```

Comment:
Do we need 3 for these?

Reply:
2 probably would do, but I feel better with 3